docs/POLYFILLS.md

```diff
@@ -0,0 +1,117 @@
+# Using Polyfills
+
+In the case you would like to use Internet Explorer, you might be interested in polyfills. 
+We don't use them by default in our application, but we've created a way to do so in three easy steps. 
```

Comment:
"Stark doesn't use any polyfills by default, but in case you need them, you can follow the steps mentioned below."

Reply:
Sorry about the abusive usage of "we", I don't know why I wrote that like this :(